edkrepo/common/clone_utilities.py

```diff
@@ -57,6 +67,25 @@ def generate_clone_cmd(repo_to_clone, workspace_dir, args=None, cache_path=None)
         except AttributeError:
             pass
 
+    try:
+        if repo_to_clone.treeless:
+            clone_cmd_args['treeless repo'] = '--filter=tree:0'
+    except AttributeError:
+        pass
+    try:
+        if repo_to_clone.blobless:
+            clone_cmd_args['blobless repo'] = '--filter=blob:none'
+    except AttributeError:
+        pass
+
+    if len(active_filters) > 1:
+        raise edkrepo_exception.EdkrepoInvalidConfigOptionException(humble.CONFLICTING_PARTIAL_CLONE)
+    elif len(active_filters) == 1:
+        clone_cmd_args['treeless repo'] = ''
+        clone_cmd_args['blobless repo'] = ''
+    elif 'treeless repo' in clone_cmd_args and 'blobless repo' in clone_cmd_args:

```

Comment:
Since we add an entry to the active filters list for each filter this case will be caught by if len(active_filters) > 1